resources/assets/js/admin/components/users/UserDetails.vue

```diff
@@ -2,20 +2,11 @@
 	<div class="user-details">
 		<wnl-text-loader v-if="isLoading" />
 		<div v-else>
-			<div class="user-details__head">
-				<p>#{{user.id}}</p>
-				<p class="user-details__head__name">{{user.full_name}}</p>
-				<span
-					v-for="role in user.roles"
-					:key="role.name"
-					class="tag"
-					:style="{backgroundColor: getColourForStr(role.name)}"
-				>
-					{{role.name}}
-				</span>
-				<p>Dołączył/-a: {{dateCreated}}</p>
-			</div>
-
+			<wnl-header
+				:text="`#${user.id} ${user.full_name}`"
+				:back-to="{ name: 'users' }"
+				sub-text="Edytor użytkonwnika"
```

Comment:
```suggestion
				sub-text="Edytor użytkownika"
```
Although I'm not sure if we should call it an editor. It's a general view with multiple purposes. I'd just use:
```suggestion
				sub-text="Użytkownik"
```